tests/test_util.py

```diff
@@ -32,6 +33,15 @@ def test_fuzz_human_readable_boolean_correct_string(answer: bool) -> None:
         assert str_answer == "No"
 
 
+@given(check_status_list=st.lists(st.booleans()))
+@pytest.mark.fuzz
+def test_total_amount_passed(check_status_list: list[bool]):
+    stats = util.total_amount_passed(check_status_list)
+
+    assert constants.markers.Zero <= stats[2] <= constants.markers.Percent_Multiplier
```

Comment:
Would you mind providing more context as to the magic numbers issue, what was causing it, and why you chose this fix? That reasoning could help with determining suitability. In the meantime, I will also do some of my own research.

Reply:
@boulais01 When linting with Ruff, you will come across errors with specific non-zero integer numbers being listed as 'magic numbers'. The idea behind this is that individual integers should be made into constant variables so that if I wanted to change every instance of this specific number, I would only have to change one variable rather then potentially thousands of instances. 

My question here is more about the format as I have never used them before. It looks in place to me, but any advice would be appreciated if applicable.

Reply:
I notice you have a constant for zero, but specify the issue is with non-zero integers? Either way, the formatting looks right to me.

Reply:
This is fine with me. I wanted to also point out that there is some debate about whether or not constants are acceptable inside of a test suite! Some people still follow ruff's rules in the test suite while others think that the constants in this file are actually better because they make the tests more explicit. What do you both think @boulais01 and @CalebKendra?